e2e-env/shop-build.sh

```diff
@@ -23,9 +25,19 @@ tunnelId="${TUNNEL_ID}"
 # Ping de l'URL
 appUrl="https://$psDomain"
 
+#Verifie qu'on a bien deux version de shop en multistore
+if [ "$profile" = "multistore" ]; then
+  if [ -z "$secondeShopVersion" ]; then
+    echo "❌ En mode multistore, tu dois fournir une deuxième version de shop."
+    exit 1
+  fi
+else
+  secondeShopVersion=""
+fi
+
 # Exécution de la commande make
 makeFilePath='./'
-makeCommand="make -C $makeFilePath docker-build PS_ACCOUNTS_VERSION=$psAccountsVersion PS_VERSION=$shopVersion PS_DOMAIN=$psDomain TUNNEL_ID=$tunnelId TUNNEL_SECRET=$tunnelSecret ACCOUNT_TAG=$accountTag"
+makeCommand="make -C $makeFilePath docker-build PS_ACCOUNTS_VERSION=$psAccountsVersion PS_VERSION=$shopVersion SECONDE_PS_VERSION=$secondeShopVersion PS_DOMAIN=$psDomain TUNNEL_ID=$tunnelId TUNNEL_SECRET=$tunnelSecret ACCOUNT_TAG=$accountTag PROFILE=$profile"
```

Comment:
```suggestion
makeCommand="make -C $makeFilePath docker-build PS_ACCOUNTS_VERSION=$psAccountsVersion PS_VERSION=$shopVersion PS_VERSION_SHOP2=$shopVersionShop2 PS_DOMAIN=$psDomain TUNNEL_ID=$tunnelId TUNNEL_SECRET=$tunnelSecret ACCOUNT_TAG=$accountTag PROFILE=$profile"
```

or

```suggestion
makeCommand="make -C $makeFilePath docker-build PS_ACCOUNTS_VERSION=$psAccountsVersion PS_VERSION=$shopVersion PS_VERSION_SECOND_SHOP=$shopVersionSecondShop PS_DOMAIN=$psDomain TUNNEL_ID=$tunnelId TUNNEL_SECRET=$tunnelSecret ACCOUNT_TAG=$accountTag PROFILE=$profile"
```

seems better

Reply:
corrected